core/internals.h

```diff
@@ -61,7 +61,7 @@
 #include <stddef.h>
 #include <stdbool.h>
 
-#include "er-coap-13/er-coap-13.h"
+#include "../coap/er-coap-13/er-coap-13.h"
```

Comment:
coap was added to the include directories. I don't think it is necessary to add the ../coap part of the path here also.